src/components/transactions/FilterModal.tsx

```diff
@@ -0,0 +1,393 @@
+                  filterState.inProgressMaxAmount === 10
+                ) {
+                  filterDispatch({ type: 'CLEAR_AMOUNT_FILTERS' });
+                } else {
+                  filterDispatch({
+                    type: 'SET_AMOUNT',
+                    minAmount: 0,
+                    maxAmount: 10,
+                  });
+                }
+              }}
+              minWidth={73}
+              margin={10}
+            >
+              <Body1Text>0-10</Body1Text>
+            </FilterTag>
+
+            <FilterTag
+              isSelected={
+                filterState.inProgressMinAmount === 11 &&
+                filterState.inProgressMaxAmount === 20
+              }
+              onPress={() => {
+                if (
+                  filterState.inProgressMinAmount === 11 &&
+                  filterState.inProgressMaxAmount === 20
+                ) {
+                  filterDispatch({ type: 'CLEAR_AMOUNT_FILTERS' });
+                } else {
+                  filterDispatch({
+                    type: 'SET_AMOUNT',
+                    minAmount: 11,
+                    maxAmount: 20,
+                  });
+                }
+              }}
+              minWidth={73}
+              margin={10}
+            >
+              <Body1Text>11-20</Body1Text>
+            </FilterTag>
+
+            <FilterTag
+              isSelected={
+                filterState.inProgressMinAmount === 21 &&
+                filterState.inProgressMaxAmount === 50
+              }
+              onPress={() => {
+                if (
+                  filterState.inProgressMinAmount === 21 &&
+                  filterState.inProgressMaxAmount === 50
+                ) {
+                  filterDispatch({ type: 'CLEAR_AMOUNT_FILTERS' });
+                } else {
+                  filterDispatch({
+                    type: 'SET_AMOUNT',
+                    minAmount: 21,
+                    maxAmount: 50,
+                  });
+                }
+              }}
+              minWidth={73}
+              margin={10}
+            >
+              <Body1Text>21-50</Body1Text>
+            </FilterTag>
+
+            <FilterTag
+              isSelected={
+                filterState.inProgressMinAmount === 51 &&
+                !filterState.inProgressMaxAmountIsSet
+              }
+              onPress={() => {
+                if (
+                  filterState.inProgressMinAmount === 51 &&
+                  !filterState.inProgressMaxAmountIsSet
+                ) {
+                  filterDispatch({ type: 'CLEAR_AMOUNT_FILTERS' });
+                } else {
+                  filterDispatch({
+                    type: 'SET_AMOUNT',
+                    minAmount: 51,
+                    maxAmount: null,
+                  });
+                }
+              }}
+              minWidth={73}
+            >
+              <Body1Text>50+</Body1Text>
+            </FilterTag>
+          </OneLine>
```

Comment:
Is it possible to select multiple amount filters? Ex: viewing all invoices between $0-20 by selecting both the `0-10` and `10-20` options. Honestly might be too complex idk.

Reply:
I think it's better to keep as is (minimumAmount & maxAmount) because then future devs just have to call the dispatch with these new amounts if they want to add a new filter option. If we enable multi-select, we would have to hardcode every option into the dispatch, and future devs would have to change this on top of the front-end.